sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_applied.py

```diff
@@ -1,11 +1,11 @@
 from collections import namedtuple
 
 
-def test_token_batch_handler_post_batch_applied(token_batch_handler, utxo_cache):
+def test_token_batch_handler_post_batch_applied(utxo_batch_handler, utxo_cache):
```

Comment:
A correct test name is `test_utxo_batch_handler_post_batch_applied`

Reply:
fixed